demo/package-lock.json

```diff
@@ -1,8283 +1,8 @@
 {
   "name": "demo",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@meghedi dont revert the lockfileVersion

Reply:
Done.